app/code/Magento/Catalog/Model/ResourceModel/Category/Collection.php

```diff
@@ -58,6 +60,60 @@ class Collection extends \Magento\Catalog\Model\ResourceModel\Collection\Abstrac
      */
     protected $_loadWithProductCount = false;
 
+    /**
+     * Core store config
+     *
+     * @var \Magento\Framework\App\Config\ScopeConfigInterface
+     */
+    protected $_scopeConfig;
```

Comment:
Please use `private` visibility. The core team avoids `protected` for new properties. Also, new properties shouldn't have a name starting with an underscore (this is part of [PSR-2](http://www.php-fig.org/psr/psr-2/)).